rdrf/rdrf/static/js/session_timeout_manager.js

```diff
@@ -24,12 +39,29 @@ function ModalSessionNotifier(options) {
             const minutesLeft = Math.round(secondsLeft / ONE_MINUTE_IN_SECONDS);
             timeoutText = interpolate(ngettext("%s minute", "%s minutes", minutesLeft), [minutesLeft]);
         } else {
-            timeoutText = interpolate(ngettext("%s second", "%s seconds", secondsLeft), [secondsLeft]);
+            const roundedSecondsLeft = Math.round(secondsLeft);
+            timeoutText = interpolate(ngettext("%s second", "%s seconds", roundedSecondsLeft), [roundedSecondsLeft]);
         }
 
         $sessionTimeoutModal.find(settings.sessionExpiryTimeoutSelector).text(timeoutText);
     }
 
+    function countdownTimer() {
+        const now = new Date()
+        const secondsLeftInSession = (sessionExpiryTimestamp.getTime() - now.getTime()) / 1000;
```

Comment:
This can cause `Uncaught TypeError: Cannot read properties of undefined (reading 'getTime')`.
Replicate steps:
(1) When session expiring modal pop up, click 'Continue working'
(2) Go to another page and then come back, above error will display in console.

Reply:
Great catch Lexie. Should be fixed in the latest commit.